flit/upload.py

```diff
@@ -59,13 +59,13 @@ def get_repositories(file="~/.pypirc"):
     return repos
 
 
-def get_repository(name=None, cfg_file="~/.pypirc"):
```

Comment:
I'm a bit surprised by the random extra changes - reordering and renaming parameters, removing default values. It doesn't really matter, and as long as this is all working, it's OK, but it's a good idea in general to keep your changes focused on the specific thing you're trying to change. More changes means it's more work to review. :wink:

Reply:
Sorry about that. 

1. I renamed the parameters to keep it mostly consistent with what we see in the interface.
1. Removed default values so they can have an only source of truth to improve maintainability in case we need to change the default value and/or debug.
1. Reordered some function in the test so it could be used in a previous test.

Please let me know if you don't think any of those changes are necessary and I would be glad to revert! :)

Reply:
I kind of like default values even if it involves some repetition, because it's an easy way to see what sort of value is expected, without having to go and look at other layers of code. And I don't think the default is likely to change. :smiley: 

E.g. here, I can see that `cfg_file` is a string path, that it needs `~` to be expanded to the home directory, and that it refers to a .pypirc config file rather than some other type of config. That's a lot of useful information - more than you're likely to have in type annotations - for a very small bit of extra code.

So maybe we can put the default values back. More generally, I'd say "if it ain't broke, don't fix it" :wink: , but there's no need to revert all the changes now.

Reply:
Got it, no worries! I can add the default values back later today if you wish.